docs/docs/01-ibc/06-channel-upgrades.md

```diff
@@ -136,6 +136,11 @@ The state will change to `FLUSHCOMPLETE` once there are no in-flight packets lef
 
 All other parameters will remain the same during the upgrade handshake until the upgrade handshake completes. When the channel is reset to `OPEN` on a successful upgrade handshake, the relevant fields on the channel end will be switched over to the `UpgradeFields` specified in the upgrade.
 
+> NOTE: due to the addition of new channel states, packets can still be received and processed in both `FLUSHING` and `FLUSHCOMPLETE` states.
+> Packets can also be acknowledged in the  `FLUSHING` state. Application developers should consider these new states
```

Comment:
Should we explicitly add that acknowledging will not be possible in `FLUSHCOMPLETE`, since all the packets sent from that channel end should have been flushed?

Reply:
Receiving allowed in both cases. Ack and Timeout only during FLUSHING